packages/graphql-yoga/src/plugins/resultProcessor/push.ts

```diff
@@ -10,7 +10,12 @@ export function processPushResult(
   result: ResultProcessorInput,
   fetchAPI: FetchAPI,
 ): Response {
-  const timeoutInSeconds = 12
+  let timeoutInSeconds = 12
+
+  // for testing the pings, reduce the timeout
+  if (process.env.NODE_ENV === 'test') {
```

Comment:
```suggestion
  if (globalEnv.process?.env?.NODE_ENV === 'test') {
```
Just to prevent `process` is not defined errors in some envs

Reply:
https://github.com/dotansimha/graphql-yoga/pull/2410/commits/53839651bc9207da2cbf36f07ec0fff7c51f71af